src/scss/components/_card.scss

```diff
@@ -0,0 +1,70 @@
+.card {
+  box-shadow: 0 0.2rem 0.2rem 0 rgba(0, 0, 0, 0.14),
+    0 0.3rem 0.1rem -0.2rem rgba(0, 0, 0, 0.12), 0 0.1rem 0.5rem 0 rgba(0, 0, 0, 0.2);
+  position: relative;
+  margin: 0.5rem 0 1rem 1rem;
+  background-color: #fff;
+  -webkit-transition: -webkit-box-shadow 0.25s;
+  transition: -webkit-box-shadow 0.25s;
+  transition: box-shadow 0.25s;
+  transition: box-shadow 0.25s, -webkit-box-shadow 0.25s;
+  border-radius: 0.3rem;
+  overflow: hidden;
+  
+  &--sm {
+    width: 27rem;
+    max-height: 60%;
+    overflow: hidden;
+  }
+  
+  &--md {
+    width: 35rem;
+  }
+  
+  &--lg {
+    width: 45rem;
+  }
+
+  &__image {
+    position: relative;
+    max-height: 60%;
+    width: 100%;
+    overflow: hidden;
+
+    img {
```

Comment:
a ideia é a &__image ser uma div que circula a imagem ou colocar essa classe na propria tag img?

se for a ultima nao precisa usar esse seletor da tag